tools/mo/openvino/tools/mo/back/ConvolutionNormalizer.py

```diff
@@ -33,15 +33,15 @@ def resolve_convolution_with_group(node: Node, group: int, ir_version: str):
         # TODO rewrite this transformation to generate a shape-computing sub-graph. Ticket 62076
```

Comment:
do we still need to resolve anything?

Reply:
Thanks for noticing this. We needed but now I implemented resolving for all cases not only for cases with dynamic channels but even when weights are dynamic.